### DISCUSSION TITLE
react-color - Warning: Prop `style` did not match

### DISCUSSION THREAD

**Discussion by eboook** [vercel/next.js#11704] [Help]
![Screenshot](https://i.stack.imgur.com/zpbG3.png "Screenshot")

I'm getting a warning when I use react-color in my index.js

**Index.js:**

```javascript 
import SketchPicker from 'react-color';

const index = () => {
    return <SketchPicker />
}
export default index;
```

**Comment by Timer** [comment]
Please open this bug on their issue tracker. It's not related to Next.js!
